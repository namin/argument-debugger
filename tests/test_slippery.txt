Marriage should be restricted to one man and one woman. Gay people should not be allowed to get married because if you allow two men or two women to get married, then why wouldn't you allow a man to marry a goat or a chair or 15 chickens?