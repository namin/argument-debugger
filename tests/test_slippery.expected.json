{
  "must_find": ["slippery_slope"],
  "may_find": ["unsupported_premise"],
  "must_not_find": ["circular", "missing_link", "false_dichotomy"]
}